README.md

```diff
@@ -31,4 +31,10 @@ helm repo add openbao https://openbao.github.io/openbao-helm
 helm install openbao openbao/openbao
 ```
 
+Alternatively you can use the OCI based helm chart as well:
+
+```
+helm install oci://ghcr.io/openbao/charts/openbao
```

Comment:
I'd say switch up the order. We can directly use the OCI based helm chart and "altternatively" use the `openbao.github.io` (github-pages) one.

Reply:
I didt not know about that. We will keep this order for now. Thanks @Nerkho